File: TaskNoteBook/src/main/java/DisplayNotes.java
/**
 * Displays notes.
 */
public class DisplayNotes implements ICommand {
  private NoteBookProvider receiver;

  /**
   * Constructor for this class.
   *
   * @param receiver used by command pattern
   */
  public DisplayNotes(NoteBookProvider receiver) {
    this.receiver = receiver;
  }

  /**
   * Displays notes.
   */
  @Override
  public void execute() {
    receiver.displayNotes();
  }
}

File: NoteBookTask/NoteBookTask/DisplayNotes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NoteBookTask
{
    class DisplayNotes : ICommand
    {
        private NoteBookProvider receiver;

        public DisplayNotes(NoteBookProvider receiver)
        {
            this.receiver = receiver;
        }

        public void Execute()
        {
            receiver.DisplayNotes();
        }

        public void Undo()
        { }
    }
}
